src/styles/blocks/product.css

```diff
@@ -0,0 +1,82 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  width: 160px;
+  height: 134px;
+}
+
+.product__disc {
+  display: inline-block;
```

Comment:
I think if the parent element has `display: flex` all of its children are treated as `inline-block` automatically, hence this rule is probably redundant.

Reply:
Done